.travis.yml

```diff
@@ -94,6 +94,12 @@ script:
    travis_wait 30 valgrind --error-exitcode=42 ./tests 16 &&
    travis_wait 30 valgrind --error-exitcode=42 ./exhaustive_tests;
    fi
+ - if [ -n "$BENCH" ]; then
```

Comment:
`BENCH` is never set

Reply:
oh I broke it when I replaced the precompiled macro with the env.
Thanks!

Reply:
Done